Makefile

```diff
@@ -43,4 +43,8 @@ clean:
 clobber: clean
 	rm -f $(CSI_PROTO) $(CSI_PROTO).tmp
 
-.PHONY: clean clobber
+# check generated files for violation of standards
+check: $(CSI_PROTO)
+	awk '{ if (length > 72) print NR, $$0 }' $? | diff - /dev/null
```

Comment:
Why 72 and not 80?

Reply:
The CONTRIBUTING guide says:
> Please wrap the code at 72 characters.

.. so that's what this PR enforces.

Reply:
Ack, we should revisit that decision.